### DISCUSSION TITLE
Elementary OS 8 & Windows 10 dual boot problem

### DISCUSSION THREAD

**Discussion by PavloMuravyov** [elementary/triage#634] [Q&A] [answered]
Hello, everyone!
I just tried to install EOS8 , and after installing get some issue. I can`t enable dual boot. I  did all installation steps like in previous EOS versions: custom install with disk partition configurating. After installing tried to update grub and made some changes in grub file.

now it looks like it:

GRUB_DEFAULT=0
GRUB_TIMEOUT_STYLE=menu
GRUB_TIMEOUT=10
GRUB_RECORDFAIL_TIMEOUT=0
GRUB_DISTRIBUTOR=`lsb_release -d -s 2> /dev/null || echo Debian`
GRUB_CMDLINE_LINUX_DEFAULT="quiet splash"
GRUB_CMDLINE_LINUX=""
GRUB_DISABLE_OS_PROBER=false

Also my disk "tree" (Windows are installed on sdb disk),

sda             8:0    0 465.8G  0 disk  
└─sda1          8:1    0 465.8G  0 part  
sdb             8:16   0 111.8G  0 disk  
└─sdb1          8:17   0 111.8G  0 part  
nvme0n1       259:0    0 223.6G  0 disk  
├─nvme0n1p1   259:1    0   512M  0 part  /boot/efi
├─nvme0n1p2   259:2    0    18G  0 part  
│ └─cryptswap 252:0    0    18G  0 crypt [SWAP]
└─nvme0n1p3   259:3    0 205.1G  0 part  /

After some searching i didn`t found any working solvers for my problem! 

Thank you!

**Comment by vjr** [comment]
What is the actual problem? Are you by any chance using AMD GPU? See https://github.com/orgs/elementary/discussions/633 essentially just remove the "splash" keyword from your grub config...

**Reply by PavloMuravyov** [reply]
No any problems with loading EOS . All is OK. Problem issue is miss grub selector (bween 2 Win and EOS ) while starting PC.

I tried your sollution. No any changes for my problem. Thank you

**Comment by vjr** [comment, marked as answer]
OK I see, you should set `GRUB_RECORDFAIL_TIMEOUT=10` this also to 10 not 0 then run `sudo update-grub` - does that fix it for  you? It should match the `GRUB_TIMEOUT=10` setting basically.

**Reply by PavloMuravyov** [reply]
Thank you. It works!
